src/snowflake/cli/api/console/console.py

```diff
@@ -10,6 +10,7 @@
 
 PHASE_STYLE: Style = Style(color="grey93", bold=True)
 STEP_STYLE: Style = Style(color="grey89", italic=True)
+INFO_STYLE: Style = Style(color="black")
```

Comment:
Thought we agreed to alias step, although I'm not sure why step is italic TBH

Reply:
I thought about it, and am not convinced that the style for step should apply to info. Hence did not go the alias route.